src/platform/web/ui/session/room/UnknownRoomView.js

```diff
@@ -29,6 +30,11 @@ export class UnknownRoomView extends TemplateView {
                 onClick: () => vm.join(),
                 disabled: vm => vm.busy,
             }, vm.i18n`Join room`),
+            t.br(),
+            t.if(vm => vm.checkingPreviewCapability, t => t.div({className: "checkingPreviewCapability"}, [
```

Comment:
```suggestion
            t.if(vm => vm.checkingPreviewCapability, t => t.div({className: "UnknownRoomVIew__PreviewIndicator"}, [
```
Following [BEM](https://getbem.com/naming/)